packages/styling/tsconfig.json

```diff
@@ -1,25 +1,23 @@
 {
   "compilerOptions": {
```

Comment:
Why did you get rid of baseUrl? IIRC it may still be needed for certain things.

Reply:
added back!